<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameLobby</class>
 <widget class="QFrame" name="FrameLobby">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::Box</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTreeWidget" name="treeWidget_battles">
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>People</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="frame_3">
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_play_red">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Play</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTreeWidget" name="treeWidget_blues">
        <property name="indentation">
         <number>5</number>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="headerDefaultSectionSize">
         <number>70</number>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Score</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QTreeWidget" name="treeWidget_reds">
        <property name="indentation">
         <number>5</number>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="headerDefaultSectionSize">
         <number>70</number>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Score</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_play_blue">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Play</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QFrame" name="frame_4">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLineEdit" name="lineEdit_battle_id">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>261</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_battle_name">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>231</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_battle_type">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>20</y>
           <width>51</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
